mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/enricher/ContentParameterDeclarationEnricher.java

```diff
@@ -92,6 +94,6 @@ private void configureDsl(ParameterDeclaration p) {
   }
 
   private List<ParameterDeclaration> getContentParameters(List<ParameterDeclaration> parameters) {
-    return parameters.stream().filter(p -> p.getRole() != BEHAVIOUR).collect(toList());
+    return parameters.stream().filter(p -> p.getRole() == PRIMARY_CONTENT || p.getRole() == CONTENT).collect(toList());
```

Comment:
This code change is just a better way of expressing what a content parameter is right?

Reply:
right. The way it was before it would break should we add a new role